src/main/java/com/joyent/manta/monitor/MantaOperationContext.java

```diff
@@ -42,7 +42,7 @@
     public static final String RESPONSE_TIMES_KEY = "responseTime";
     public static final String EXCEPTION_KEY = "exception";
     public static final String STOPWATCH_KEY = "stopwatch";
-    public static final String REQUEST_PUT_HISTOGRAMS_KEY = "requestPutHistograms";
+    public static final String REQUEST_PUT_HISTOGRAMS_MAP = "requestPutHistograms";
```

Comment:
Well, this is a key. It makes sense to have the suffix `_KEY`.

Reply:
Okay will change it back to ``` REQUEST_PUT_HISTOGRAMS_KEY```